CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ v1.8.0 (unreleased)
     if the response body was empty.
 -   Adds support for the `UnrecognizedProcedureError` error and error checker,
     indicating that the router was unable to find a handler for the request.
+-   Exposes a `InboundMiddleware` function on the dispatcher to allow applying
+    middleware to custom handlers.
```

Comment:
Consider,

> Adds an accessor to Dispatcher which provides access to the inbound middleware used by that Dispatcher.

We don't need to state how/why this might be used.